bears/python/VultureBear.py

```diff
@@ -1,25 +1,56 @@
-import re
 from shutil import which
 
 from coalib.bears.GlobalBear import GlobalBear
 from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
 
 
+class Item(str):
+
+    def __new__(cls, name, typ, filename, lineno):
+        item = str.__new__(cls, name)
+        item.typ = typ
+        item.filename = filename
+        item.lineno = lineno
+        return item
+
+
+class Unused(Item):
+
+    def __new__(cls, item, message):
+        item.message = message
+        return item
+
+
+def retrieve_unused(filenames):
+    import vulture
```

Comment:
That is because you re-assign a different value to `vulture` later:

> vulture = vulture.Vulture()

Bam!  python decides you want a local variable, and not use the global variable.